scripts/py_matter_idl/matter_idl/zapxml/handlers/handlers.py

```diff
@@ -110,7 +110,7 @@ def GetNextProcessor(self, name: str, attrs):
             self._event.readacl = AttrsToAccessPrivilege(attrs)
             return BaseHandler(self.context, handled=HandledDepth.SINGLE_TAG)
         elif name.lower() == 'description':
-            return BaseHandler(self.context, handled=HandledDepth.ENTIRE_TREE)
+            return DescriptionHandler(self.context, self._event)
```

Comment:
@bzbarsky-apple `class Event` does not have a description attribute: https://github.com/project-chip/connectedhomeip/blob/master/scripts/py_matter_idl/matter_idl/matter_idl_types.py#L162

This will likely work because duck typing, however it does not seem correct.

Maybe the other change saying "description handles entire tree" is what we want as errors were tied to commands and other things already supported? I still am not clear why the errors though as it should have been marked as handled...

Reply:
https://github.com/project-chip/connectedhomeip/pull/29114 is handling this.